packages/perseus/src/types.js

```diff
@@ -75,6 +75,22 @@ export type Version = {|
 
 export type EditorMode = "edit" | "preview" | "json";
 
+export type ChoiceState = {|
+    selected: boolean,
+    crossedOut: boolean,
+    highlighted: boolean,
+    rationaleShown: boolean,
+    correctnessShown: boolean,
+    previouslyAnswered: boolean,
+    readOnly: boolean,
+|};
+
+export type RadioChoiceWithMetadata = {|
```

Comment:
If this is only used in widget.jsx would it make more sense to put it in that file?

Reply:
Yes, I think it might, especially because if we do another custom type with PerseusRadioChoice and we were putting them all here, it might get repetitive/excessive. If we put the custom types in the file that uses it, then we will have fewer imports there and less code in this communal types file. Maybe if we use this type again, instead of importing it from widget, we would then move it into the types file?

Reply:
Oh, so it looks like radio.jsx also uses this type, so I will leave it in types.js :)

Reply:
Note that `widget.jsx` is really poorly named.  It is actually the main widget implementation of the `radio` widget (it lives at `packages/perseus/src/widgets/radio/widget.jsx`).  

So I think it would make alot of sense to at least move this type somewhere inside of `packages/perseus/src/widgets/radio/` instead of it looking like a type that can be used outside of the radio widget implementation. :)

Reply:
I knew I'd be too late, but still wanted to make the note. 

I'd say do this in its own PR. Smaller, focused PRs are generally easier and faster to review and land. 😄